src/b_SideMenu/SideMenu.jsx

```diff
@@ -41,9 +41,9 @@ class SideMenu extends React.Component<Props> {
                 role="button"
                 key={item.id}
                 to={item.path}
-                className={`ac-side-menu-item ${
+                className={classNames(`ac-side-menu-item ${
```

Comment:
Please, change this template string to use classnames lib APIinstead
```${ this.state.currentItem === item.id ? 'active' : ''}```

```suggestion
                classNames(['ac-side-menu-item', {'active': this.state.currentItem === item.id}, classList])
```